docs/sandbox-permissions-reference.rst

```diff
@@ -0,0 +1,51 @@
+Sandbox Permissions
+===================
+
+Sandbox permissions can be configured from an application manifest file (see :doc:`manifests`). They can also be set with the ``build-finish``, ``run`` and ``override`` commands.
+
+The following list includes many of the most useful permission options. A complete list can be viewed using ``flatpak build-finish --help``.
+
+===================================================  ===========================================
+``--nofilesystem=...``                               Undo some of the above
```

Comment:
Missing `--filesystem`

Reply:
I've removed --nofilesystem - I think it's better explained as part of the page on debugging.